ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -905,6 +907,21 @@ private Builder applyHoleskyNetworkDefaults() {
               "enr:-LS4QG0uV4qvcpJ-HFDJRGBmnlD3TJo7yc4jwK8iP7iKaTlfQ5kZvIDspLMJhk7j9KapuL9yyHaZmwTEZqr10k9XumyCEcmHYXR0bmV0c4gAAAAABgAAAIRldGgykGm32XQEAXAAAAEAAAAAAACCaWSCdjSCaXCErK4j-YlzZWNwMjU2azGhAgfWRBEJlb7gAhXIB5ePmjj2b8io0UpEenq1Kl9cxStJg3RjcIIjKIN1ZHCCIyg");
     }
 
+    private Builder applyEphemeryNetworkDefaults() {
+      return applyTestnetDefaults()
+          .constants(EPHEMERY.configName())
+          .startupTimeoutSeconds(120)
+          .trustedSetupFromClasspath(MAINNET_TRUSTED_SETUP_FILENAME)
+          .eth1DepositContractDeployBlock(0)
+          .defaultInitialStateFromUrl(
+              "https://checkpoint-sync.ephemery.ethpandaops.io/eth/v2/debug/beacon/states/finalized")
+          .customGenesisState(
+              "https://github.com/ephemery-testnet/ephemery-genesis/releases/latest/download/genesis.ssz")
+          .discoveryBootnodes(
+              // TEKU bootnode
+              "enr:-Iq4QNMYHuJGbnXyBj6FPS2UkOQ-hnxT-mIdNMMr7evR9UYtLemaluorL6J10RoUG1V4iTPTEbl3huijSNs5_ssBWFiGAYhBNHOzgmlkgnY0gmlwhIlKy_CJc2VjcDI1NmsxoQNULnJBzD8Sakd9EufSXhM4rQTIkhKBBTmWVJUtLCp8KoN1ZHCCIyk");
```

Comment:
we won't have a teku bootnode i don't expect, but this would be where we'd list any ephemery network discovery nodes that we can use during startup

Reply:
> we won't have a teku bootnode i don't expect, but this would be where we'd list any ephemery network discovery nodes that we can use during startup

Yes. This is the one Ephemery is using on Teku. I will remove the comment to avoid the confusion. That was how it was labelled on the Ephemery repo

Reply:
oh ok - that's fine... just for clarity we generally put 'teku bootnode' when its run by us specifically so that we can track it, so maybe just remove the comment in that case, and we'll look at it as a black box...